### PR TITLE
RHDEVDOCS-3389 - removal of references to the monitoring Grafana component and UI

### PR DESCRIPTION
**Pull request by bburt-rh**
Summary: This PR removes all references in the documentation to the Grafana component and Grafana UI because Grafana is no longer included in the monitoring stack and has been removed from OCP. 

This PR only removes or rewrites content referring to the Grafana component or UI. The PR does not add any new content.

- Aligned team: DevTools
- For branches:  4.11
- Jira: https://issues.redhat.com/browse/RHDEVDOCS-3389
- Direct link to doc previews: 
- - http://file.rdu.redhat.com/bburt/RHDEVDOCS-3389-document-grafana-removal-from-monitoring/monitoring/monitoring-overview.html#default-monitoring-components_monitoring-overview
- - http://file.rdu.redhat.com/bburt/RHDEVDOCS-3389-document-grafana-removal-from-monitoring/monitoring/monitoring-overview.html#understanding-the-monitoring-stack_monitoring-overview
- - http://file.rdu.redhat.com/bburt/RHDEVDOCS-3389-document-grafana-removal-from-monitoring/monitoring/configuring-the-monitoring-stack.html#configurable-monitoring-components_configuring-the-monitoring-stack
- - http://file.rdu.redhat.com/bburt/RHDEVDOCS-3389-document-grafana-removal-from-monitoring/monitoring/accessing-third-party-monitoring-apis.html
- - http://file.rdu.redhat.com/bburt/RHDEVDOCS-3389-document-grafana-removal-from-monitoring/logging/troubleshooting/cluster-logging-log-store-status.html#ref_cluster-logging-elasticsearch-cluster-status_cluster-logging-elasticsearch
- - http://file.rdu.redhat.com/bburt/RHDEVDOCS-3389-document-grafana-removal-from-monitoring/monitoring/configuring-the-monitoring-stack.html#moving-monitoring-components-to-different-nodes_configuring-the-monitoring-stack
- - http://file.rdu.redhat.com/bburt/RHDEVDOCS-3389-document-grafana-removal-from-monitoring/logging/cluster-logging-dashboards.html#cluster-logging-dashboards-access_cluster-logging-dashboards
- SME review: @simonpasquier (approved)
- QE review: @juzhao (approved) @lihongyan1 (approved)
- Peer review: @rolfedh (approved)

### CODE REVIEW

**Comment by juzhao** [comment]
cc: @lihongyan1

**Comment by juzhao** [comment]
from http://file.rdu.redhat.com/bburt/RHDEVDOCS-3389-document-grafana-removal-from-monitoring/monitoring/monitoring-overview.html
`Understanding the monitoring stack` section, grafana should be removed from graph, we need to use a updated architecture graph

**Comment by juzhao** [comment]
http://file.rdu.redhat.com/bburt/RHDEVDOCS-3389-document-grafana-removal-from-monitoring/monitoring/configuring-the-monitoring-stack.html
should remove grafana from `Table 1. Configurable monitoring components`

File: modules/understanding-quick-start-elements.adoc
Comment (juzhao) [inline modules/understanding-quick-start-elements.adoc:150]: no need to remove, since it's related to openshift-monitoring grafana

File: modules/nodes-nodes-viewing-listing.adoc
Comment (juzhao) [inline modules/nodes-nodes-viewing-listing.adoc:173]: since we removed grafana pod, we also need to update
`Non-terminated Pods:                      (13 in total)`
to 
`Non-terminated Pods:                      (12 in total)`

File: modules/understanding-quick-start-elements.adoc
Comment (simonpasquier) [reply modules/understanding-quick-start-elements.adoc:150]: Agreed the Grafana mentioned here isn't related to the Grafana shipped by the cluster monitoring operator :)

**Review by simonpasquier — COMMENTED** [review]
no more comments than what @juzhao already mentioned

File: modules/understanding-quick-start-elements.adoc
Comment (bburt-rh) [reply modules/understanding-quick-start-elements.adoc:150]: fixed

File: modules/nodes-nodes-viewing-listing.adoc
Comment (bburt-rh) [reply modules/nodes-nodes-viewing-listing.adoc:173]: Good catch. Fixed.

**Comment by bburt-rh** [comment]
> http://file.rdu.redhat.com/bburt/RHDEVDOCS-3389-document-grafana-removal-from-monitoring/monitoring/configuring-the-monitoring-stack.html should remove grafana from `Table 1. Configurable monitoring components`

Fixed. Please verify at http://file.rdu.redhat.com/bburt/RHDEVDOCS-3389-document-grafana-removal-from-monitoring/monitoring/configuring-the-monitoring-stack.html#configurable-monitoring-components_configuring-the-monitoring-stack

**Review by simonpasquier — COMMENTED** [review]
lgtm

File: modules/infrastructure-moving-monitoring.adoc
Comment (rolfedh) [inline modules/infrastructure-moving-monitoring.adoc:10]: Hi Brian. The current text is fine. But I started looking for ways to make a couple of improvements:
- Resolving the difference in terminology between the title and the abstract: "solution" vs. "stack"
- Using active instead of passive voice: "is deployed" > "deploys"
- Bringing the lede, "moving it components," to the fore.
- Providing a better term than "machines."

How about something like:
```suggestion
To <summary of benefits>, you can redeploy the Prometheus Cluster Monitoring stack to infrastructure nodes by using a custom config map.
```
OR, if the benefits are obvious or indeterminate:
```suggestion
To redeploy the Prometheus Cluster Monitoring stack to infrastructure nodes, you can create and apply a custom config map.
```

Comment (rolfedh) [reply modules/infrastructure-moving-monitoring.adoc:10]: Then, expand on the abstract in a separate paragraph:
```
By default, the Prometheus Cluster Monitoring stack includes Prometheus, Thanos Querier, and AlertManager. is deployed to provide cluster monitoring. The Cluster Monitoring Operator manages this stack.
```

File: scalability_and_performance/scaling-cluster-monitoring-operator.adoc
Comment (rolfedh) [inline scalability_and_performance/scaling-cluster-monitoring-operator.adoc:9]: Maybe use the same term for the stack here:
```suggestion
{product-title} exposes metrics that the Cluster Monitoring Operator collects and stores in the the Prometheus Cluster Monitoring stack. As an administrator, you can view dashboards for system resources, containers, and components metrics in the {product-title} web console by navigating to *Observe* -> *Dashboards*.
```

**Review by rolfedh — APPROVED** [review]
Left some optional suggestions.

File: scalability_and_performance/scaling-cluster-monitoring-operator.adoc
Comment (bburt-rh) [reply scalability_and_performance/scaling-cluster-monitoring-operator.adoc:9]: I'm leaving this as-is. We don't capitalize the word "monitoring" in the docs, and we don't use the phrase "cluster monitoring stack" anywhere else. Generally, we just refer to the "monitoring stack" and saying `Prometheus-based monitoring stack` isn't inconsistent with this practice.

File: modules/infrastructure-moving-monitoring.adoc
Comment (bburt-rh) [reply modules/infrastructure-moving-monitoring.adoc:10]: Using your suggestions, I rewrote the paragraph to read:

```
The monitoring stack includes multiple components, including Prometheus, Thanos Querier, and Alertmanager.
The Cluster Monitoring Operator manages this stack.
To redeploy the monitoring stack to infrastructure nodes, you can create and apply a custom config map.
```

File: modules/monitoring-accessing-third-party-monitoring-uis.adoc
Comment (lihongyan1) [inline modules/monitoring-accessing-third-party-monitoring-uis.adoc:10]: Suppose this part can be deleted, all the third-party UI are disabled now.

File: monitoring/accessing-third-party-monitoring-uis-and-apis.adoc
Comment (lihongyan1) [inline monitoring/accessing-third-party-monitoring-uis-and-apis.adoc:11]: All the third-party UI are disabled now, grafana is disabled for both API and UI.  Maybe better just write API only here

You can access web service API from the command line interface (CLI) for the following third-party monitoring components: : Alertmanager, Prometheus, Thanos Ruler, and Thanos Querier.

File: scalability_and_performance/scaling-cluster-monitoring-operator.adoc
Comment (rolfedh) [reply scalability_and_performance/scaling-cluster-monitoring-operator.adoc:9]: That makes sense. I had made that recommendation based on the opening paragraph, which included "Prometheus Cluster Monitoring," and the following instances elsewhere in the openshift-docs repo:

Results for "Prometheus Cluster Monitoring" in "openshift-docs/*.adoc"

/home/rdlugyhe/openshift-docs/whats_new/new-features.adoc:
  82:  |Prometheus cluster monitoring


/home/rdlugyhe/openshift-docs/modules/cluster-logging-deploy-cli.adoc:
  48:  with metrics, you should configure the Prometheus Cluster Monitoring stack to scrape metrics from the `openshift-operators-redhat`


/home/rdlugyhe/openshift-docs/modules/infrastructure-moving-monitoring.adoc:
  10:  By default, the Prometheus Cluster Monitoring stack, which contains Prometheus, Grafana, and AlertManager,


/home/rdlugyhe/openshift-docs/modules/machine-config-daemon-metrics.adoc:
  8:  provides a set of metrics. These metrics can be accessed using the Prometheus Cluster Monitoring stack.

File: monitoring/accessing-third-party-monitoring-uis-and-apis.adoc
Comment (bburt-rh) [reply monitoring/accessing-third-party-monitoring-uis-and-apis.adoc:11]: Revised so that we no longer mention third-party web UIs at all -- only APIs for Alertmanager, Prometheus, Thanos Ruler, and Thanos Querier.

**Comment by bburt-rh** [comment]
@lihongyan1 I've revised per your feedback. Please review again and approve if ready. The preview can be seen at: http://file.rdu.redhat.com/bburt/RHDEVDOCS-3389-document-grafana-removal-from-monitoring/monitoring/accessing-third-party-monitoring-apis.html

File: modules/monitoring-accessing-third-party-monitoring-web-service-apis.adoc
Comment (simonpasquier) [reply modules/monitoring-accessing-third-party-monitoring-web-service-apis.adoc:13]: hmm `monitoring-alertmanager-edit` should allow access to all Alertmanager API endpoints. I'll test it on my side.

Comment (bburt-rh) [reply modules/monitoring-accessing-third-party-monitoring-web-service-apis.adoc:13]: Lmk what you find @simonpasquier. If the Alertmanager API is not available to users with the `monitoring-alertmanager-edit` role, then I think we should log a separate issue against this content and fix it in a separate PR. This fix would be out of scope for the current PR, which only intends to remove references to Grafana.

Comment (juzhao) [reply modules/monitoring-accessing-third-party-monitoring-web-service-apis.adoc:13]: please see the whole page, there is one Prerequisite
```
The following example commands show how to query the service API receivers for Alertmanager. This example requires that the associated user account be bound against the monitoring-alertmanager-edit role in the openshift-monitoring namespace and that the account has the privilege to view the route. This access only supports using a Bearer Token for authentication.
```
https://github.com/openshift/openshift-docs/pull/41384#discussion_r807687996 
mentioned the monitoring-alertmanager-edit role is not enough to see the alertmanager APIs, we changed to as it is now

Comment (lihongyan1) [reply modules/monitoring-accessing-third-party-monitoring-web-service-apis.adoc:13]: > please see the whole page, there is one Prerequisite
> 
> ```
> The following example commands show how to query the service API receivers for Alertmanager. This example requires that the associated user account be bound against the monitoring-alertmanager-edit role in the openshift-monitoring namespace and that the account has the privilege to view the route. This access only supports using a Bearer Token for authentication.
> ```
> 
> [#41384 (comment)](https://github.com/openshift/openshift-docs/pull/41384#discussion_r807687996) mentioned the monitoring-alertmanager-edit role is not enough to see the alertmanager APIs, we changed to as it is now

Even given the host, a user with `monitoring-alertmanager-edit` can't access alert receivers api

Comment (simonpasquier) [reply modules/monitoring-accessing-third-party-monitoring-web-service-apis.adoc:13]: having said that it shouldn't block this PR since it's an orthogonal issue to the change.

Comment (bburt-rh) [reply modules/monitoring-accessing-third-party-monitoring-web-service-apis.adoc:13]: I agree with @simonpasquier that this is an orthogonal issue outside the scope of this PR, and I don't think it should block this PR from being merged. @lihongyan1 - if you disagree, please state your reasons. But if you both agree that this isn't a blocker, I'll need you both to approve this PR as-is, so that I can get it merged. Then, one of us can write up a new docs Jira issue describing the problem with the Alertmanager sample provided and instructions on how to address the issue. Thanks!

Comment (lihongyan1) [reply modules/monitoring-accessing-third-party-monitoring-web-service-apis.adoc:13]: filed a bug to track the issue
https://bugzilla.redhat.com/show_bug.cgi?id=2102601

Comment (bburt-rh) [reply modules/monitoring-accessing-third-party-monitoring-web-service-apis.adoc:13]: Thanks all. Marking this as resolved.

**Review by simonpasquier — COMMENTED** [review]
lgtm

**Pull request merged**
